lib/src/sai_redis_switch.cpp

```diff
@@ -1,19 +1,12 @@
 #include "sai_redis.h"
+#include "sairedis.h"
```

Comment:
yes, i will change that later, new one will be distributes in the package, previous one is internal